src/lib/consensus/proof_of_stake.ml

```diff
@@ -491,48 +491,78 @@ module Vrf = struct
   module Threshold = struct
     open Bignum_bigint
 
-    let of_uint64_exn = Fn.compose of_int64_exn UInt64.to_int64
-
     (* TEMPORARY HACK FOR TESTNETS: c should be 1 otherwise *)
-    let c_int = 2
+    let c : [`One | `Two] = `Two
```

Comment:
We put this hack in to counteract the decreased fill rate when using more proposers. Now that we're using the right threshold function it's not necessary any more.

Reply:
+1